dev-libs/libical/libical-2.0.0.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="An implementation of iCalendar protocols and data formats"
+DESCRIPTION="Libical is an Open Source implementation of the iCalendar \
+protocols and protocol data units. The iCalendar specification describes how \
+calendar clients can communicate with calendar servers so users can store \
+their calendar data and arrange meetings with other users."
+HOMEPAGE="http://libical.github.io/libical/"
+COPYRIGHT="1999-2015 Eric Busboom"
+LICENSE="MPL v1.0
+	GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/libical/libical/releases/download/v$portVersion/libical-$portVersion.tar.gz"
+CHECKSUM_SHA256="654c11f759c19237be39f6ad401d917e5a05f36f1736385ed958e60cf21456da"
+PATCHES="libical-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	libical$secondaryArchSuffix = $portVersion compat >= 2
+	lib:libical$secondaryArchSuffix = $portVersion compat >= 2
+	lib:libical_cxx$secondaryArchSuffix = $portVersion compat >= 2
+	lib:libicalss$secondaryArchSuffix = $portVersion compat >= 2
+	lib:libicalss_cxx$secondaryArchSuffix = $portVersion compat >= 2
+	lib:libicalvcal$secondaryArchSuffix = $portVersion compat >= 2
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	icu$secondaryArchSuffix
```

Comment:
I think you should replace:
```
	icu$secondaryArchSuffix
```
by
```
	lib:libicui18n$secondaryArchSuffix
	lib:libicuuc$secondaryArchSuffix

```
But maybe we should wait for a confirmation from another contributor.